python/py-notebook_shim/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-notebook_shim
+version             0.2.3
+revision            0
+categories-append   devel
+license             BSD
+supported_archs     noarch
+platforms           {darwin any}
+
+python.versions     37 38 39 310 311
+python.pep517       yes
+python.pep517_backend hatch
+
+maintainers         openmaintainer
+
+description         This project provides a way for JupyterLab and other frontends to switch to Jupyter Server for their Python Web application backend
```

Comment:
this should ideally be a shorted description that fits in the 80 character limit